releng/release-tool/release-tool.go

```diff
@@ -308,7 +308,21 @@ func (r *releaseData) makeTag(branch string) error {
 
 	r.generateReleaseNotes()
 
-	_, err := gitCommand("-C", r.repoDir, "tag", "-s", r.tag, "-F", r.releaseNotesFile)
+	if !r.dryRun {
```

Comment:
I think `dryRun` is not needed here, this is only modifying a local copy of the repo. Only the push below should be guarded by `dryRun`.

Reply:
Makes sense.